spyder/plugins/application/widgets/status.py

```diff
@@ -116,11 +116,20 @@ def _on_click(self):
     # ---- Public API
     # -------------------------------------------------------------------------
     def show_appeal(self):
-        if self._appeal_dialog is None:
-            self._appeal_dialog = InAppAppealDialog(self)
-
-        if not self._appeal_dialog.isVisible():
-            self._appeal_dialog.show()
+        from qtpy import QtModuleNotInstalledError
+        try:
+            if self._appeal_dialog is None:
+                self._appeal_dialog = InAppAppealDialog(self)
+
+            if not self._appeal_dialog.isVisible():
+                self._appeal_dialog.show()
+        except QtModuleNotInstalledError:
+            # QtWebEngineWidgets is optional
+            # So just open the URL in the default browser
+            # See https://github.com/spyder-ide/spyder/pull/23703
```

Comment:
```suggestion
            # QtWebEngineWidgets is optional, so just open the URL in the
            # default browser.
            # See spyder-ide/spyder#24905 for the details.
```

- Improve comment a bit.
- Reference this PR instead (with the format we use for that) because the other one only has a CI commit now.